tsconfig.json

```
@@ -1,21 +1,21 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es2020",
```

```suggestion
    "target": "es5",
```
using babel allow us to have a greater compatibility, we should not target a version which is the new.